.travis.yml

```diff
@@ -22,7 +22,7 @@ install:
   - docker-compose -f docker-compose.yml up -d
   - git clone https://github.com/Human-Connection/Nitro-Backend.git ../Nitro-Backend
   - git -C "../Nitro-Backend" checkout $TRAVIS_BRANCH || echo "Branch \`$TRAVIS_BRANCH\` does not exist, falling back to \`master\`"
-  - docker-compose -f ../Nitro-Backend/docker-compose.yml up -d
+  - docker-compose -f ../Nitro-Backend/docker-compose.yml -f ../Nitro-Backend/docker-compose.travis.yml up -d
```

Comment:
@roschaefer where does the magic `docker-compose.travis.yml` come from?

Reply:
https://github.com/Human-Connection/Nitro-Backend/blob/master/docker-compose.travis.yml